### ISSUE TITLE
Add Admin Actions to 'SendCavcRemandProcessedLetterTask'

### ISSUE THREAD

**Issue by araposo-tistatech** [department-of-veterans-affairs/caseflow#15286]
## User or job story
User story: As a Litigation Support user, I need the ability to add admin actions to the send 90 day letter task, so that appropriate actions on the case that are necessary are completed. 

## Acceptance criteria
- [ ] This feature should be accessible to the following user groups: CAVC Litigation Support
- [ ] Include screenshot(s) in the Github issue if there are front-end changes
- [ ] Update documentation: CAVC Remand
- [ ] Task action "add admin action" is added to SendCavcRemandProcessedLetterTask
- [ ] The following actions are listed on the submit admin actions page
  - [ ] IHP – use current routing rules (Routed to vlj support) - Blocks distribution
  - [ ] Missing Hearing Transcripts – Routed to the Hearing Branch (Routed to Transcription Team)  - Blocks distribution
  - [ ] FOIA – use current routing rules (Routed to privacy team) - Blocks distribution
  - [ ] Translation – use current routing rules (Routed to Translation Team) - Blocks Distribution to Judge (current routing rules) 
  - [ ] POA clarification – Routed to CAVC Litigation Support Team, Blocks completion of the Send 90 Day Letter task, task is placed on hold queue until POA Clarification task is completed
- [ ] Provide Instructions and context for this action appears on the submit admin actions page
- [ ] Verify IHP, Missing Hearing Transcripts, & POA Clarification continue to block Distribution to VLJs (they currently do)

**Comment by geronimoramos** [comment]
@araposo-tistatech @
- For lit support admin actions: leave in notes, not individual tasks -> make this clear in QRG and training
- for tasks that are sent to other teams provide for them in the admin actions dropdown

future evaluation: if needed for reporting purposes can single out those admin actions

**Comment by yoomlam** [comment]
See #15548
Task tree:
```ruby
                                                      ┌─────────────────────────────────────────────────────────────────────────────────────────────┐
Appeal 9 (evidence_submission) ────────────────────── │ ID   │ STATUS   │ ASGN_BY                │ ASGN_TO                │ UPDATED_AT              │
└── RootTask                                          │ 24   │ on_hold  │                        │ Bva                    │ 2020-11-03 21:30:55 UTC │
    └── DistributionTask                              │ 25   │ on_hold  │                        │ Bva                    │ 2020-11-03 21:30:55 UTC │
        ├── EvidenceSubmissionWindowTask              │ 26   │ assigned │                        │ MailTeam               │ 2020-11-03 21:30:55 UTC │
        └── CavcTask                                  │ 3369 │ assigned │                        │ Bva                    │ 2020-11-04 19:36:31 UTC │
            └── SendCavcRemandProcessedLetterTask     │ 3378 │ on_hold  │                        │ CavcLitigationSupport  │ 2020-11-04 23:01:26 UTC │
                └── SendCavcRemandProcessedLetterTask │ 3383 │ assigned │ CAVC_LIT_SUPPORT_ADMIN │ CAVC_LIT_SUPPORT_USER1 │ 2020-11-04 23:01:26 UTC │
                                                      └─────────────────────────────────────────────────────────────────────────────────────────────┘

```

**Comment by hschallhorn** [comment]
_[what is this chart?](https://github.com/department-of-veterans-affairs/caseflow/issues/11533)_
```bash
1 | 
2 | |
3 | ||||
5 | |||||
8 | 
```

Original work involves (if using the existing add-admin-action modal):
* limit admin actions to the 5 listed
* add the associated task to be directly under DistributionTask
* change POA logic to route to CAVC Litigation Support Team

Alternative:
* add 5 different task actions, 4 of them using the "send to team" modal, 1 new for POA
* (Do not use the add-admin-action modal)

Let's go with the alt here. Feel free to break this up

1. Implement for one task action
2. Determine if it is viable to be worked on by other engineers at the same time or can be done in one PR

Loop in design early for copy on send to team modal.

#### Why 2?
 - Will be easy to replicate once we implement one

#### Why 3?
* Send To Team is a very familiar task action for the echo team

#### Why 5?
* Amount of work 
* Lets of testing to do

**Comment by araposo-tistatech** [comment]
Confirmed with Mike and John the Missing Hearing Transcripts should be routed to the Transcription Team.

**Comment by hschallhorn** [comment]
closed in #15607
